ambry-messageformat/src/main/java/com.github.ambry.messageformat/MessageFormatRecord.java

```diff
@@ -518,6 +741,57 @@ public static DeleteRecord deserializeDeleteRecord(CrcInputStream crcStream)
   }
 
   /**
+   *  - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   * |         |           |                      |            |
+   * | version |   size    |  blob encryption key |     Crc    |
+   * |(2 bytes)| (4 bytes) |  (n bytes)           |  (8 bytes) |
+   * |         |           |                      |            |
+   *  - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+   *  version                - The version of the blob key
+   *
+   *  size                   - The size of the blob key
+   *
+   *  blob encryption key    - The blob encryption key in bytes
+   *
+   *  crc                    - The crc of the blob key record
+   *
+   */
+  public static class BlobEncryptionKey_Format_V1 {
+    public static final int Blob_Encryption_Key_Size_Field_In_Bytes = 4;
```

Comment:
use Integer.BYTES

Reply:
Again - I have used the convention in this file.

Reply:
For atleast the new code we write, we use this notation. Would be nice if we can start following it.